upgrading/upgrade-operator.adoc

```diff
@@ -7,7 +7,10 @@ include::modules/common-attributes.adoc[]
 toc::[]
 
 [role="_abstract"]
-Upgrades through the {rh-rhacs-first} Operator are performed automatically or manually, depending on the *Update approval* option you chose at installation.
+Upgrades through the {rh-rhacs-first} Operator are performed automatically or manually, depending on the *Update approval* option you chose at installation. Upgrading from version 3.74 to 4.x requires some additional steps that are described on this page. Upgrading within 4.y versions requires no additional steps.
```

Comment:
I suggest stick to `4.x` for consistency unless you meant to highlight the difference between `4.x` and `4.y` in this sentence.

```suggestion
Upgrades through the {rh-rhacs-first} Operator are performed automatically or manually, depending on the *Update approval* option you chose at installation. Upgrading from version 3.74 to 4.x requires some additional steps that are described on this page. Upgrading within 4.x versions requires no additional steps.
```